src/components/Worker.tsx

```diff
@@ -0,0 +1,111 @@
+interface WorkerProps {workerInfo: {id: number, firstName: string, lastName:string,
+     title:string, country: string, city:string, birthDate:Date, imageUrl:string, time: string}}
+
+const Worker = ({workerInfo}: WorkerProps) => {
+    const age: number = new Date().getFullYear() - workerInfo.birthDate.getFullYear();
+    console.log(workerInfo.birthDate);
+    return (
+        <>
```

Comment:
This component is really big. Split it into 2-3 components (before doing so, tell me how you decided to split it into - think about code readability and reusability in other parts of the website, even though it won't be needed for this part of the training).